contracts/Staking.sol

```diff
@@ -664,6 +689,28 @@ contract Staking is Pauser, Whitelist {
         emit DelegationUpdate(_valAddr, delAddr, validator.tokens, delegator.shares, -int256(_tokens));
     }
 
+    function _removeFrom(
+        address _addr,
+        address[] storage _addrArray,
+        int256 _index
+    ) private {
+        uint256 index = uint256(_index);
```

Comment:
I am a bit confused about why we need to have `int256 _index` and convert it to `uint256` here?  Looks like we could just use `uint256 _index` directly.

(Note that `_index` is a stack variable so we could safely change it)

Reply:
right.